client/state/terms/selectors.js

```diff
@@ -5,6 +5,23 @@ import get from 'lodash/get';
 import values from 'lodash/values';
 
 /**
+ * Returns true if a network request is in-progress for the specified site ID,
+ * taxonomy pair, or false otherwise.
+ *
+ * @param  {Object}  state    Global state tree
+ * @param  {Number}  siteId   Site ID
+ * @param  {String}  taxonomy Taxonomy Slug
+ * @return {Boolean}          Whether request is in-progress
+ */
+export function isRequestingSiteTaxonomyTerms( state, siteId, taxonomy ) {
+	if ( ! state.terms ) {
```

Comment:
We can always assume `state.terms` (and `state.terms.requesting`) will exist.

Background: When the Redux store is initialized, it fires a `@@redux/INIT` action, and ensures that all reducers in the tree [return a non-`undefined` initial state](https://github.com/reactjs/redux/blob/9045a0c1732ab266a1ff09926420f266e1a74d65/src/utils/combineReducers.js#L22-L26).